src/components/SideNav.tsx

```diff
@@ -115,7 +115,7 @@ const NavLink: React.FC<IPropsNavLink> = ({ item, path }) => {
           key={item.id}
           animate={isOpen ? "open" : "closed"}
           variants={innerLinksVariants}
-          initial="closed"
+          initial={isOpen ? "open" : "closed"}
```

Comment:
Remove default "closed", and align with logic of the `isOpen` boolean

Reply:
<img width="456" alt="image" src="https://github.com/ethereum/ethereum-org-fork/assets/54227730/5cb3981d-db5b-4d65-b320-795f638268cb">

Should fix any glitching when the page loads, and be open to this on first paint

Reply:
On the doc's root I still see these collapsed https://deploy-preview-213--ethereum-org-fork.netlify.app/developers/docs